versions.props

```diff
@@ -42,14 +43,13 @@ org.mockito:* = 5.11.0
 org.slf4j:* = 2.0.17
 org.wildfly.common:wildfly-common = 1.7.0.Final
 jakarta.ws.rs:jakarta.ws.rs-api = 4.0.0
+jakarta.validation:jakarta.validation-api = 3.0.0
 
 # dependency-upgrader:OFF
-# Avoid forcing consumers to upgrade jackson
-com.fasterxml.jackson.*:jackson-* = 2.12.3
```

Comment:
This was outdated and effectively useless given the versions.lock pin was already bumped